Resources/Prototypes/DeltaV/Entities/Objects/Misc/subdermal_implants.yml

```diff
@@ -15,3 +15,42 @@
       tags:
         - SubdermalImplant
         - BionicSyrinxImplant
+
+- type: entity
+  parent: BaseSubdermalImplant
```

Comment:
inherit storage implanter also these should be in a new file in the deltav folder